generic-sensor/generic-sensor-tests.js

```diff
@@ -300,8 +350,7 @@ function runGenericSensorTests(sensorType) {
       123,
       {},
       "",
-      true,
-      undefined
```

Comment:
and here `undefined` is valid input as `referenceFrame` has a default value (based on https://heycam.github.io/webidl/#es-dictionary)